cardano-api/test/Test/Cardano/Api/TxBody.hs

```diff
@@ -284,3 +372,30 @@ buildMintValue = \case
 
 tests :: TestTree
 tests = $testGroupGenerator
+
+
+-- | hack to work around https://github.com/input-output-hk/cardano-ledger-specs/pull/2529
+newtype Show' a = Show' a
+  deriving newtype Eq
+
+instance Show a => Show (Show' a) where
```

Comment:
I don't understand. Why do we need this?

Reply:
This was needed for debugging. Removed.